File: src/main/java/com/example/matchteam/domain/board/Board.java
package com.example.matchteam.domain.board;

import lombok.Builder;
import lombok.Data;

import java.sql.Timestamp;

@Data
public class Board {
    private Long id;
    private String title;
    private String content;
    private int viewCnt;
    private Timestamp createdDate;
    private int statusId;

    @Builder
    public Board(Long id, String title, String content, int viewCnt, Timestamp createdDate, int statusId) {
        this.id = id;
        this.title = title;
        this.content = content;
        this.viewCnt = viewCnt;
        this.createdDate = createdDate;
        this.statusId = statusId;
    }
}

File: MatchingTeam/app/src/main/java/com/example/matchingteam/domain/board/Board.kt
package com.example.matchingteam.domain.board

import com.google.gson.annotations.SerializedName
import java.sql.Timestamp

data class Board(
    @SerializedName("id")
    var id: Long,
    @SerializedName("title")
    var title: String,
    @SerializedName("content")
    var content: String,
    @SerializedName("viewCnt")
    var viewCnt: Int,
    @SerializedName("createdDate")
    var createdDate: Timestamp,
    @SerializedName("statusId")
    var statusId: Int
)